File: py101/easy3/6.py
# https://launchschool.com/exercises/fddf66ac

def main():

    def get_prompt():
        
        while True:
            noun = input("Enter a noun: ")
            verb = input("Enter a verb: ")
            adjective = input("Enter an adjective: ")
            adverb = input("Enter an adverb: ")
            
            print()
            tell_story(noun, verb, adjective, adverb)
            print()

    def tell_story(noun, verb, adjective, adverb):
        print(f"Do you {verb} your {adjective} {noun} {adverb}? That's hilarious!")
        print(f"The {adjective} {noun} {verb}s {adverb} over the lazy {noun}.")
        print(f"The {noun} {adverb} {verb}s up to Joe's {adjective} turtle.")

    get_prompt()

if __name__ == "__main__":
    main()

File: js210/simpleProblems/easy2/6.js
// https://launchschool.com/exercises/85e333ca

'use strict';

let relSync = require('readline-sync');
let noun = relSync.question('Enter a noun: ');
let verb = relSync.question('Enter a verb: ');
let adjective = relSync.question('Enter an adjective: ');
let adverb = relSync.question('Enter an adverb: ');

console.log(`Do you ${verb} your ${adjective} ${noun} ${adverb}? That's hilarious!`);
